packages/terra-search-field/tests/jest/SearchField.test.jsx

```diff
@@ -213,12 +213,16 @@ describe('Auto Search', () => {
   });
 
   it('uses standard timeout for search delay when not provided', () => {
+    jest.useFakeTimers();
+    jest.spyOn(global, 'setTimeout');
```

Comment:
Can you add `jest.restoreAllMocks()` at the end of the function?

Reply:
Added in the afterAll section.